app/mailers/app_mailer.rb

```diff
@@ -1,11 +1,16 @@
 class AppMailer < ActionMailer::Base
   def welcome_email(user)
     @user = user
-    mail from: 'jojojoship@gmail.com', to: user.email, subject: "Welcome to MyFlix!"
+    mail from: 'info@myflix.com', to: user.email, subject: "Welcome to MyFlix!"
```

Comment:
You can set a `default from: "info@myflix.com"` above all mail methods. Like a callback.